_config.yml

```diff
@@ -93,6 +93,8 @@ collections:
   cards:
     output: true
     permalink: /card/:name/
+  irl:
```

Comment:
Minor: To avoid cluttering the top-level directory, I would try to move this under some other directory.